controllers/ocsinitialization/ocsoperator_config.go

```diff
@@ -0,0 +1,126 @@
+package ocsinitialization
```

Comment:
@malayparida2000 i dont think this commit is required at all. you can set the required configuration in the Rook ENV in csv-merger.go

Reply:
never mind, i didnt notice that we cannot add runtime calculation to csi-merger.go

Reply:
Yes, And Jose suggested let's have a configmap specifically for OCS-Operator which can be useful in the future also, And let's keep the existing rook-ceph-operator-config CM just for customer use.